test/unit/article/article.spec.js

```diff
@@ -224,120 +230,139 @@ describe('Test for article', () => {
-      firstUserToken = firstUser.body.token;
+        expect(response.body.status).to.be.equal('Fail');
+        expect(response.status).to.equal(404);
+        expect(response.body.message).to.be.deep.equals('Article not found');
+      });
     });
 
-    before(async () => {
-      const secondUser = await request.post(loginEndpoint).send(successfulLogin2);
-      secondUserToken = secondUser.body.token;
-    });
+    describe('Test for deleting articles', () => {
+      before(async () => {
+        const firstUser = await request.post(loginEndpoint).send(loginData);
+        firstUserToken = firstUser.body.token;
+      });
 
-    it('should delete an article', async () => {
-      const response = await request
-        .delete(`${articlesEndpoint}/${slug}`)
-        .set('Authorization', firstUserToken);
-      expect(response.body.status).to.be.equal('Success');
-      expect(response.status).to.equal(200);
-      expect(response.body.message).to.be.deep.equals('Article deleted successfully');
-    });
+      before(async () => {
+        const secondUser = await request.post(loginEndpoint).send(successfulLogin2);
+        secondUserToken = secondUser.body.token;
+      });
 
-    it('should not delete an article', async () => {
-      const response = await request
-        .delete(`${articlesEndpoint}/${secondSlug}`)
-        .set('Authorization', 'jdkfdjkfdkf');
+      it('should delete an article', async () => {
+        const response = await request
+          .delete(`${articlesEndpoint}/${slug}`)
+          .set('Authorization', firstUserToken);
+        expect(response.body.status).to.be.equal('Success');
+        expect(response.status).to.equal(200);
+        expect(response.body.message).to.be.deep.equals('Article deleted successfully');
+      });
 
-      expect(response.status).to.equal(403);
-    });
+      it('should not delete an article', async () => {
+        const response = await request
+          .delete(`${articlesEndpoint}/${secondSlug}`)
+          .set('Authorization', 'jdkfdjkfdkf');
 
-    it('should not delete an unknown article', async () => {
-      const response = await request
-        .delete(`${articlesEndpoint}/hfd-fdfd-vwdcas-325525`)
-        .set('Authorization', firstUserToken);
+        expect(response.status).to.equal(403);
+      });
 
-      expect(response.body.status).to.be.equal('Fail');
-      expect(response.status).to.equal(404);
-      expect(response.body.message).to.be.deep.equals('Article not found');
+      it('should not delete an unknown article', async () => {
+        const response = await request
+          .delete(`${articlesEndpoint}/hfd-fdfd-vwdcas-325525`)
+          .set('Authorization', firstUserToken);
+
+        expect(response.body.status).to.be.equal('Fail');
+        expect(response.status).to.equal(404);
+        expect(response.body.message).to.be.deep.equals('Article not found');
+      });
     });
   });
 });
```

Comment:
HI Juwon, I like what you have done with your feature, but going through your code, I realized there is no way an author could edit or remove tags they have already created. I think you might want to look into that.

Reply:
Hmm, Nice feedback @frostyblok, I didn't think of that. I'll implement it and give you feedback

Reply:
OK cool!

Reply:
Hey Juwon, your ability to implement feedback swiftly is really nice. Good job!

Reply:
thanks @frostyblok 😃